corehq/motech/fhir/tasks.py

```diff
@@ -563,7 +562,7 @@ def sync_all_appointments_domain(domain):
         for appointment in epic_appointment_records:
             appointment_resource = appointment.get('resource')
             appointment_id = appointment_resource.get('id')
-            if appointment_id and appointment_id not in appointment_fhir_ids:
+            if appointment_id and appointment_id not in appointment_map.keys():
```

Comment:
You don't actually need to use `.keys()` here since dicts support key lookup by default